server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -705,7 +753,9 @@ void removeAndDeletePoolsOnOtherOwners(List<Pool> existingPools, Pool pool) {
 
             // quantity has changed. delete any excess entitlements from pool
             if (updatedPool.getQuantityChanged()) {
-                this.deleteExcessEntitlements(existingPool);
+                List<Pool> existingPools = new ArrayList<Pool>();
+                existingPools.add(existingPool);
+                this.deleteExcessEntitlements(existingPools);
```

Comment:
In the future when you need a quick list like this, you can use Arrays.asList to wrap an element (or elements).